src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -29,6 +29,15 @@ import org.scalatest.FunSuite
 /* Implicits */
 import TestSQLContext._
 
+private[avro] object DirectoryDeletion {
```

Comment:
What I needed was a method to recursively delete directories so that we don't use FileUtils. This method is used by AvroSuite, WriteBenchmark and AvroFileGenerator. Since all of those are a part of testing infrastructure, AvroSuite would be a good place to put it. I would have made it a static method, but since I can't, I just have a separate object that has this method. Is this considered hacky?